src/git-sync

```diff
@@ -46,6 +46,7 @@ function skippable {
 
 
 function steps {
+  echo_if_true "change_directory $(git_root)" "$IN_SUB_FOLDER"
```

Comment:
Agreed. I'll add it to the other commands in separate PRs. I have added checkboxes to #371 to track when all commands are addressed.